docs/core/examples/guides/omnigraph-aica-bridge.md

```diff
@@ -0,0 +1,349 @@
+---
+sidebar_position: 11
+title: Interface with Isaac Sim using OmniGraph and ROS 2
+---
+
+import selector from './assets/omnigraph-aica-bridge-selector.png'; 
+import sceneCreate from './assets/omnigraph-aica-bridge-scene-create.webm'; 
+import graph from './assets/omnigraph-aica-bridge-graph.png'; 
+import addingGraph from './assets/omnigraph-aica-bridge-add-graph.webm'; 
+import runningAICA from './assets/omnigraph-aica-bridge-aica-app.webm';
+import integration from './assets/omnigraph-aica-bridge-integration.webm';
+
+# Interface with Isaac Sim using OmniGraph and ROS 2
+
+This guide walks you through the steps required to set up **NVIDIA Isaac Sim** so it can interface with **AICA Core**
+using **OmniGraph** and **ROS 2**. By the end of this tutorial, you’ll have a working simulation environment in Isaac
+Sim that communicates with AICA Core over ROS 2.
+
+**NVIDIA Isaac Sim** is a high-fidelity robotics simulator built on NVIDIA Omniverse. It provides realistic physics,
+multi-sensor support, and RTX-based rendering, making it ideal for developing, testing, and validating robot software
+before deploying to real hardware.
+
+To connect Isaac Sim with AICA Core, we will use **ROS 2 Bridge**, an extension that lets Isaac Sim publish and
+subscribe to ROS 2 topics and services. Through this bridge, we can exchange robot state, control commands, sensor data,
+and more between the simulator and AICA Core.
+
+**OmniGraph** is a visual, node-based programming system integrated into Isaac Sim. It lets you assemble logic and data
+flows, called _Action Graphs_, by connecting pre-built nodes. OmniGraph can include ROS 2 Bridge nodes, enabling
+communication between Isaac Sim and ROS 2 without writing code.
+
+This interface can be used in two main ways:
+
+1. **Control robots in Isaac Sim from AICA Core:** In this mode, AICA Core sends motion commands and control inputs to
+   the simulated robot via ROS 2, and Isaac Sim returns robot state and sensor feedback. From AICA’s point of view, the
+   robot in Isaac Sim behaves just like real hardware, which is ideal for validating control algorithms before deploying
+   them to physical robots.
+
+2. **Visualize robots in Isaac Sim from AICA Core:** Here, the robot is controlled by AICA Core (e.g., in a production
```

Comment:
I think this lacks precision. In both modes, the *robot* is controlled by "an AICA application" or "an application running in AICA Studio". The difference is that 1) the robot is once virtual and once real and 2) that we feed back the state or not